console/src/pages/Model/ModelListCard.tsx

```diff
@@ -40,7 +40,10 @@ export default function ModelListCard() {
                 data={
                     modelsInfo.data?.list.map((model) => {
                         return [
-                            <TextLink key={model.id} to={`/projects/${projectId}/models/${model.id}`}>
+                            <TextLink
+                                key={model.id}
+                                to={`/projects/${projectId}/models/${model.id}/versions/${model.version?.id}/overview`}
```

Comment:
when `ModelVersionVo` is null is this link a bad address?

Reply:
We do not support deleting the resources by version, so this can not happen